classes/functions.php

```diff
@@ -142,8 +137,7 @@ public function writerLookup($comic_id) {
    *
    * @param int $series_id
    */
-  public function issuesList($series_id) {
-    $ownerID = $_SESSION ['user_id'];
+  public function issuesList($series_id, $ownerID) {
```

Comment:
Adding $ownerID variable to several functions here to allow the user to pass another username to view their collection.